src/appkit.cs

```diff
@@ -23144,9 +23144,6 @@ partial interface NSResponder {
 
 		[Mac (10, 8), Export ("quickLookWithEvent:")]
 		void QuickLook (NSEvent withEvent);
-
-		[Mac (10, 7), Export ("validateProposedFirstResponder:forEvent:")]
-		bool ValidateProposedFirstResponder (NSResponder responder, [NullAllowed] NSEvent forEvent);
```

Comment:
I guess that's the branch mix up you mentioned ?

Reply:
Correct.